mis_builder/models/mis_report_instance.py

```diff
@@ -733,7 +871,11 @@ def _compute_matrix(self):
         aep = self.report_id._prepare_aep(
             self.query_company_ids, self.currency_id)
         kpi_matrix = self.report_id.prepare_kpi_matrix()
-        for period in self.period_ids:
+        if not self.use_automatic_periods:
```

Comment:
The changes between standard periods and automatic periods is made here. I think that the whole machinery doesn't see the difference.